docs/modules/plugins/pages/plugin-sauce-labs.adoc

```diff
@@ -69,6 +69,11 @@ a|URL
 |
 |Sauce Labs REST endpoint (it must match to the selected data center)
 
+|[subs=+quotes]`*saucelabs.app-url*`
+a|URL
+|
+|Sauce Labs App Server URL, e.g. `https://app.eu-central-1.saucelabs.com` (it must match to the selected data center)
```

Comment:
it's possible to get App Server by region: https://github.com/saucelabs/saucerest-java/blob/master/src/main/java/com/saucelabs/saucerest/DataCenter.java (but it's not released yet)